games-emulation/fbalpha_libretro/fbalpha_libretro-0.2.97.42.git.recipe

```diff
@@ -0,0 +1,44 @@
+SUMMARY="A port of Final Burn Alpha to the libretro architecture"
+DESCRIPTION="Final Burn Alpha is a multi-systems Arcade Emulator most popular for \
+emulating Neo-Geo, Capcom, Konami, and Cave games. It is developed by the Final Burn \
+team and originated from FinalBurn by Dave."
+HOMEPAGE="https://github.com/libretro/fbalpha"
+COPYRIGHT="2000-2018 the Final Burn team, the libretro team"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/libretro/fbalpha/archive/099c24b166f08b671f703051de1daffb2e6d5eb7.tar.gz"
+CHECKSUM_SHA256="2ac4a4cf35aaf3380195b4804d9574be33ae52f4f95b7bd3ab8be450de5c8673"
+SOURCE_DIR="fbalpha-099c24b166f08b671f703051de1daffb2e6d5eb7"
+PATCHES="fbalpha_libretro-$portVersion.patchset"
```

Comment:
It would be nice to replace the patchset by a file in `haikuports/games-emulation/fbalpha_libretro/additional-files/` that we could name `fbalpha_libretro.info.in`.
```diff
-PATCHES="fbalpha_libretro-$portVersion.patchset"
+ADDITIONAL_FILES="fbalpha_libretro.info.in"
```
That file would be almost identical to the `fbalpha_libretro.info` file, except for one line:
```diff
-display_version = "v0.2.97.42"
+display_version = "@DISPLAY_VERSION@"
```
Then in `BUILD()` you would only need this change:
```diff
 BUILD()
 {
+	sed -e "s/@DISPLAY_VERSION@/v${portVersion%~.*}/" \
+		$portDir/additional-files/fbalpha_libretro.info.in \
+		> fbalpha_libretro.info
 	make -f makefile.libretro $jobArgs
 }
```

Reply:
Sure. Will do.